### PR TITLE
mock default_build_test_runner so unit test doesn't run build test again

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: test/unit/presubmit-tests.sh [hunk 1]
```diff
@@ -42,6 +42,10 @@ function init_test_env() {
   function check_links_in_markdown() {
     return 0
   }
+
+  function default_build_test_runner() {
+    return 0
```
Comment (adrcunha) [inline test/unit/presubmit-tests.sh]: You're not checking that the default runner was executed...

Comment (chaodaiG) [reply test/unit/presubmit-tests.sh]: Added another test for it

**Pull request merged**